testSuite.py

```diff
@@ -0,0 +1,113 @@
+from utilities import ReturnCode
+
+# timeout for pexpect. short_timeout for the cases where there should not be communication
+# and long_timeout for the cases where there should be
```

Comment:
Yes, I think it is better to put it in the main script (because if other people add testCases they don't care about the timeout). But I'm thinking that maybe it should be the same for all cases? Because if we put two different timeouts it could be seen as we have a bias about the result (?). It would take a bit more time for the execution in a couple of tests but maybe it is worthy